net-misc/passt/passt-2023.12.04.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="User-mode networking daemons for VMs and namespaces, replacement for Slirp"
+HOMEPAGE="https://passt.top/"
+
+RELEASE_COMMIT="b86afe3"
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="git://passt.top/passt"
+else
+	SRC_URI="https://passt.top/passt/snapshot/passt-${RELEASE_COMMIT}.tar.xz -> ${P}.tar.xz"
+	S="${WORKDIR}/${PN}-${RELEASE_COMMIT}"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="BSD GPL-2+"
+SLOT="0"
+IUSE="static"
+# no test suite
+RESTRICT="test"
```

Comment:
I don't think that we usually restrict test if there is no test suite. Doesn't the default `src_test()` degraded to a no-op? Otherwise, maybe define it was such?

Reply:
Just deleting that line. src_test defaults to `make check` or `make test` which doens't exists in this case